pkg/cmd/gist/view/view.go

```diff
@@ -51,6 +52,7 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
 
 	cmd.Flags().BoolVarP(&opts.Raw, "raw", "r", false, "Print raw instead of rendered gist contents")
 	cmd.Flags().BoolVarP(&opts.Web, "web", "w", false, "Open gist in the browser")
+	cmd.Flags().BoolVarP(&opts.Files, "files", "l", false, "Display filenames list from the gist")
```

Comment:
We don't need a shorthand for this flag. You can leave the shorthand argument an empty string.